forte/data/extractor/base_extractor.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
```

Comment:
The purpose it to fix this problem https://stackoverflow.com/questions/15853469/putting-current-class-as-return-type-annotation.

Reply:
This is interesting. Actually I also encountered this issue and I just sismple put a quote around the type. Not sure if I need to import future as well though.

Reply:
Unfortunately, this only work for python3.7 but not python3.6.

I think I will need to use string as python type too.